source/appModules/outlook.py

```diff
@@ -332,6 +332,24 @@ def _generateTimeRangeText(self,startTime,endTime):
 		# Translators: a message reporting the time range (i.e. start time to end time) of an Outlook calendar entry
 		return _("{startTime} to {endTime}").format(startTime=startText,endTime=endText)
 
+	@staticmethod
+	def _generateCategoriesText(appointment):
+		categories = appointment.Categories
+		if not categories:
+			return None
+		if len(categories.split("; ")) > 1:

```

Comment:
Please use ngettext in such cases, not every language has just one plural form.

Reply:
It looks like we never use ngettext in the code base. @josephsl do you have any thoughts to share here?

Reply:
I think it is something we should be using. Not sure if there are any issues preventing it's use, someone has to go first to test it out 😛